### ISSUE TITLE
osx builds fails randomly on Travis

### ISSUE THREAD

**Issue by nils-van-zuijlen** [OpenLightingProject/ola#1540]
As you may have seen, osx builds fail randomly to pass the tests on Travis. You may not see it because by restarting the build, it "fixes itself".

**Comment by nils-van-zuijlen** [comment]
For example of failing jobs see

- https://travis-ci.org/OpenLightingProject/ola/jobs/474150101
- https://travis-ci.org/nils-van-zuijlen/ola/jobs/448410753 (this one is on my copy of Travis CI)
- https://travis-ci.org/OpenLightingProject/ola/jobs/474150104
- https://travis-ci.org/OpenLightingProject/ola/jobs/481859294
- https://travis-ci.org/OpenLightingProject/ola/jobs/481859290

**Comment by peternewman** [comment]
Yes, unfortunately I was aware. I think I had a look at one point and didn't get very far.

Our buildslaves don't do it, so it's not exclusively OS X:
https://buildbot.openlighting.org/builders/buildcheck-ola-0.10-macos-x86_64-1
https://buildbot.openlighting.org/builders/buildcheck-ola-macos-x86_64-1

I normally just retry once or twice and it comes good. The Linux ones do so occasionally too, but less often.
